### ISSUE TITLE
Allow Import and Export Routing Policy at the AS Level

### ISSUE THREAD

**Issue by zimage** [respawner/peering-manager#117]
### Environment
* Peering Manager version: master

### Proposed Functionality
When peering with an organization that has several peering IP addresses, it would be convenient to be able to have policy scoped at the AS level.

### Use Case

I'm thinking that import policy like "tag routes learned from AS1234 with 65000:1234" would be best configured at the AS level, not at the peer level

**Comment by jlixfeld** [comment]
+1 however a peer level policy for session 1.2.3.4/AS1234 should probably override an AS level policy that would already have applied a policy for session 1.2.3.4 / AS1234.

**Comment by FredNeuberger** [comment]
+1 to both of you :)

**Comment by respawner** [comment]
The policy weight is not really defined by any logic in PM. I mean, it sure makes sense that IX Policy < AS Policy < Session Policy, but that logic is up to the user to be defined in the template.

That said it is interesting to have policies per AS to be defined. This kind of policies could also be used for direct peering sessions as well.

**Comment by zimage** [comment]
We are a juniper shop, so we have many import and export policies defined for each peer. Only a few of them are really different between AS peers. We have an import policy that tags all routes learned at the IX with a "we learned this route in $region" community, then we have AS-level filters like the import filter from irrpt, and we tag with a "we learned this route from $as_num" community. This is in addition to the boilerplate no-small-communities, no-bogons, etc kinds of filters.

I'm looking forward to seeing the IRR integration features once they come out. That's the real reason I'm interested in this tool. It's a lot easier to teach a tool like this to a new NOC engineer than to describe how to ssh into a machine, run irrpt_fetch and irrpt_pfxgen, copy files to the router, etc...
